tools/dragen-germline/4.3.6/dragen-germline__4.3.6.cwl

```diff
@@ -79,6 +79,22 @@ requirements:
             exit 1
           fi
 
+          # DRAGEN Multi-region Joint Detection (MRJD) is a de novo germline small variant caller for paralogous regions.
+          # MRJD is compatible with the hg38, hg19 and GRCh37 reference genomes.
+          # https://help.dragen.illumina.com/product-guides/dragen-v4.3/dragen-dna-pipeline/small-variant-calling/multi-region-joint-detection
+          #
+          # Multi Region Joint Detection (MRJD) Caller should be runs as standalone pipeline on DRAGEN™ server (not in integrated with Germline Small VC)
+          # https://support.illumina.com/content/dam/illumina-support/documents/downloads/software/dragen/release-notes/200056923_00_DRAGEN_4_3_6_Customer-Release-Notes.pdf
+          echo "run MRJD" 1>&2
```

Comment:
You can look at the if statements shown in https://github.com/umccr/cwl-ica/blob/main/tools/dragen-somatic/4.3.6/dragen-somatic__4.3.6.cwl#L93 as an example of how to nest this inside an if statement so that this component only runs when enable_mrjd is set to true,  i.e 

i.e

```
if [[ "$(get_bool_value_as_str(inputs.enable_mrjd ))" == "true" ]]; then
  # Run mrjd
  ...
fi
```